src/types/UserToken.ts

```diff
@@ -3,6 +3,7 @@ import User from './User';
 export default interface UserToken {
   id?: string;
   role?: string;
+  rolesACL?: string[];
```

Comment:
It does not seem to be used.

Reply:
Yes it is used in check of auths right away without rebuilding it on the fly whenever there is an HTTP request:

![image](https://user-images.githubusercontent.com/25266945/114101424-02a13d00-98c6-11eb-8968-4467c23a5390.png)